Product/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+class Main_category(models.Model):
+	main_category = models.CharField(max_length=45)
```

Comment:
```suggestion
	name = models.CharField(max_length=45)
```
카테고리의 이름이므로 'name'이 적당합니다.
아래 다른 클래스에서도 클래스 이름과 똑같은 속성은 가운데 의미가 이름인 것들은 전부 `name`으로 그 의미에 맞게 수정해주세요